src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -80,6 +80,10 @@ def validate_lab_vm_list(namespace):
         namespace.filters = "Properties/ownerObjectId eq '{}'".format(object_id)
 
 
+def validate_user_name(namespace):
+    namespace.user_name = "@me"
```

Comment:
REST endpoint exposes `user_name` parameter but from the cli context we are hiding them to enable scenario that it's the user's context. so that default value for that parameter will be `@me`. 

For that reason I've created validator to default. Let me know if that is not the way we should be defaulting.